app/views/CheckYourAnswersView.scala.html

```diff
@@ -34,15 +34,15 @@
 
 @layout(pageTitle = titleNoForm(messages("checkYourAnswers.title"))) {
 
-    @heading(messages("checkYourAnswers.heading"))
+    <h1 class="govuk-heading-l">@(messages("checkYourAnswers.heading"))</h1>
```

Comment:
should we add this to the component?

@(message: String, size: string ="xl")

<h1 class="govuk-heading-@size">@message</h1>

Reply:
Will do so in a follow up